src/components/blocks/layout/Header.tsx

```diff
@@ -51,17 +98,38 @@ export const Header = ({ data }: { data: PageBlocksHeader }) => {
 			)}
 		>
 			<div className='flex max-w-7xl justify-between items-center w-full p-4'>
-				{data.logo && data.logo.src && data.logo.alt && (
-					<Image src={data.logo?.src} alt={data.logo.alt} width={200} height={60} />
-				)}
+				<div className='flex items-center'>
+					{data.logo && data.logo.src && data.logo.alt && (
+						<Image src={data.logo?.src} alt={data.logo.alt} width={200} height={60} />
+					)}
+				</div>
 				{data.navLinks && (
-          <>
-            <Group className='md:w-1/2 flex justify-between' visibleFrom='sm'>
-              {items}
-            </Group>
-            <Burger opened={opened} onClick={toggle} size="sm" hiddenFrom="sm" />
-          </>
+					<>
+						<Group className='md:w-1/2 flex justify-between' visibleFrom='sm'>
+							{items}
+						</Group>
+						<Burger opened={opened} onClick={toggle} size='sm' hiddenFrom='sm' />
+					</>
 				)}
+				<div className='hidden md:flex items-center space-x-8'>
```

Comment:
So to add spacing between items you can ensure that there is horizontal padding or margin spacing around the links. In this case, since it is a button, you can put horizontal margin as padding would increase the size of the button itself so `mx-4` for example would add more spacing around the button.

However, I noticed we are using `space-x-[number]` which applies space between values hence why the first and last element are at the edge of their divs ('About' and 'Contact' are pushed to the edge of their divs). I think we can try to use just take advantage of the flex display and have each element share same [padding](https://tailwindcss.com/docs/padding) or [margin](https://tailwindcss.com/docs/margin) around itself to ensure its all consistently spaced.

Reply:
Just to share: Padding increases the size of the element itself so like a button or link would have an increased area that the user can interact with it but margin increasing would just add white space instead

Reply:
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/213081b6-a194-43ae-be6e-7dab3c90c3db">